readme.md

```diff
@@ -253,7 +253,7 @@ The data type is number and the function is `+`, the addition of two numbers.
 
 The identity value is `0` - adding `0` to any number will not change it.
 
-For something to be a monoid, it's also required that the order of operations will not affect the result:
+For something to be a monoid, it's also required that the grouping of operations will not affect the result:
```

Comment:
Array concatenation is monoidal as discussed, but the order does matter.

    [1, 2, 3].concat([4, 5]) !== [4, 5].concat([1, 2, 3])

The important rule is _associativity_, meaning grouping.